javascript/operators/import_meta_resolve.json

```diff
@@ -0,0 +1,50 @@
+{
+  "javascript": {
+    "operators": {
+      "import_meta": {
```

Comment:
This should be a subfield under `import_meta`, like all other API properties

Reply:
Sorry, I meant to name this 
```suggestion
      "import_meta_resolve": {
```

Should it be something else, though?

Reply:
Yes—you don't need to create a new file. Just add a new sub feature under `import_meta.json`.